### PR TITLE
Remove symbol table and debug info from go binary to reduce image size.

### PR DESCRIPTION
**Pull request by oilbeater**

### CODE REVIEW

File: build/backend.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ gulp.task('backend', ['package-backend'], function(doneFn) {
         '-i',
         // record version info into src/version/version.go
         '-ldflags',
-        conf.recordVersionExpression,
+        conf.recordVersionExpression + ' -w -s',
```
Comment (cheld) [inline build/backend.js]: This code is only executed during development. Not to build a release binary or docker image. I think we can keep debug information in this case.

Comment (oilbeater) [reply build/backend.js]: That's right, we should keep them for development

**Comment by floreks** [comment]
Fix error and we can merge.

https://travis-ci.org/kubernetes/dashboard/jobs/326317609

**Comment by oilbeater** [comment]
error fixed @floreks

**Pull request merged**